TypeCobol.Test/Report/ReportTestHelper.cs

```diff
@@ -73,8 +74,17 @@ public static void ParseWithNodeListenerReportCompare(string fileName, string re
                             string result = sw.ToString();
                             string expected = File.ReadAllText(output, format.Encoding);
                             TypeCobol.Test.TestUtils.compareLines(input, result, expected, PlatformUtils.GetPathForProjectFile(output));
+                            return true;
                         }
                     }
+                    else
+                    {
+                        return false;
```

Comment:
Can you return more information than just false in the case if there are diagnostics or the report is null ?
Running the test in debug just to know what happened can be prevented if the information is in the output.

Reply:
Do you want to return the Diagsnotics list?

Reply:
Not necessarily the Diagnostics list, but at least a message explaining why the test failed (e.g. "diagnostics encountered", "report not generated correctly"...)

Reply:
Is a return code enough?